Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -10,45 +10,72 @@ namespace Microsoft.MixedReality.Toolkit.Utilities.Solvers
     /// <summary>
     /// This class handles the solver components that are attached to this <see href="https://docs.unity3d.com/ScriptReference/GameObject.html">GameObject</see>
     /// </summary>
-    public class SolverHandler : ControllerFinder
```

Comment:
>ControllerFinder [](start = 33, length = 16)

Nit: Also work having a section on the migration docs for this (i.e. no longer extends ControllerFinder, here's what you do instead!)